spring-batch-infrastructure/src/test/java/org/springframework/batch/item/kafka/KafkaItemWriterTests.java

```diff
@@ -103,6 +103,19 @@ public void testBasicDelete() throws Exception {
 		verify(this.kafkaTemplate).sendDefault(items.get(1), null);
 	}
 
+	@Test
+	public void testKafkaTemplateCanBeReferencedFromSubclass() {
+		KafkaItemWriter<String, String> kafkaItemWriter = new KafkaItemWriter<String, String>() {
+			@Override
+			protected void writeKeyValue(String key, String value) {
+				this.kafkaTemplate.flush();
```

Comment:
This could be changed to `this.kafkaTemplate.sendDefault(key, value);`, because currently the parameters are not used. Do you agree?